botCommands/searchmdn.js

```diff
@@ -0,0 +1,19 @@
+const Discord = require('discord.js');
+const { registerBotCommand } = require('../botEngine');
+
+const searchmdn = {
```

Comment:
Adding a comment here, so we can keep the PR feed clean.

Reply:
TBH I think having both at the same time could be confusing (specially to new helpers). Given that we are apparently rewriting everything to ~~Rust~~ slash embed commands, I'd rather have only one. @Asartea

Reply:
For completeness, I have the same remarks about the `!google` [PR](https://github.com/TheOdinProject/odin-bot-v2/pull/388)

Reply:
Yeah preserving some of them as `!` commands is already the plan (see #297 ). I think this should be one of them, but thats of course just my opinion. @01zulfi do you have any thoughts on this?